wolfcrypt/test/test.c

```diff
@@ -36,7 +36,8 @@
     #define HAVE_WOLFCRYPT_TEST_OPTIONS
 #endif
 
-#ifdef HAVE_WOLFCRYPT_TEST_OPTIONS
+#if defined(HAVE_WOLFCRYPT_TEST_OPTIONS) || \
+    defined(WOLF_CRYPTO_CB_ONLY_ECC)
```

Comment:
I could not build due to this change.... On an embedded target we don't want ssl.h or test.h. Please remove `WOLF_CRYPTO_CB_ONLY_ECC` from here.

Reply:
removed